src/main/java/ghidrathon/interpreter/GhidrathonInterpreter.java

```diff
@@ -66,6 +68,10 @@ private GhidrathonInterpreter() throws JepException, IOException{
 		
 		// configure Python includes Path
 		config.addIncludePaths(paths);
+
+		// configure Jep stdout and stderr
+		config.redirectStdout(new WriterOutputStream(out, "UTF-8"));
+		config.redirectStderr(new WriterOutputStream(err, "UTF-8"));
```

Comment:
Do we foresee any potential encoding issues by manually specifying `UTF-8`? e.g. Windows v Linux v Mac?

Reply:
<img width="273" alt="image" src="https://user-images.githubusercontent.com/89736193/228513302-7adc1e48-a29e-4440-9c08-b476092b649f.png">

As per the apache docs Charset is a required param and specifiying it explicitly does look out for possible encoding issues

Reply:
Understood! Is there any way for us to determine the encoding at runtime? e.g. is there a method we can call on `out` or `err` to get the encoding used and pass it to `WriterOutputStream`?